js/mail.js

```diff
@@ -1306,6 +1306,45 @@ $(document).ready(function() {
 	$(document).on('change input paste keyup', '.reply-message-fields #to', Mail.UI.toggleReplyButton);
 	$(document).on('change input paste keyup', '.reply-message-body', Mail.UI.toggleReplyButton);
 
+	/**
+	* Detects pasted text by browser plugins, and other software.
+	* Check for changes in message bodies every second.
```

Comment:
Is this really a good idea performance-wise? Or is this likely to cause lags? It’s not _that_ important if the tradeoff is too big.